src/social-network-provider/twitter.com/utils/fetch.ts

```diff
@@ -1,20 +1,63 @@
 import { bioCard } from './selector'
+import { regexMatch } from '../../../utils/utils'
+import { notNullable } from '../../../utils/assert'
+import { defaultTo, join, trim } from 'lodash-es'
 
-export const resolveInfoFromBioCard = () => {
-    const avatar = bioCard()
-        .querySelector<HTMLImageElement>('img')
-        .map(x => x.src)
-        .evaluate()!
-    const userNames = bioCard()
-        .map(x => (x.children[1] as HTMLElement).innerText.split('\n'))
-        .evaluate()!
-    const bio = bioCard()
-        .map(x => (x.children[2] as HTMLElement).innerText)
-        .evaluate()!
+export const bioCardParser = () => {
+    const avatar = notNullable(
+        bioCard()
+            .querySelector<HTMLImageElement>('img')
+            .map(x => x.src)
+            .evaluate(),
+    )
+    const userNames = notNullable(
+        bioCard()
+            .map(x => (x.children[1] as HTMLElement).innerText.split('\n'))
+            .evaluate(),
+    )
+    const bio = notNullable(
+        bioCard()
+            .map(x => (x.children[2] as HTMLElement).innerHTML)
+            .evaluate(),
+    )
     return {
         avatar,
         name: userNames[0],
-        handle: userNames[1],
+        handle: notNullable(regexMatch(userNames[1], /@(.+)/)),
         bio,
     }
 }
+
+export const postContentParser = (node: HTMLElement) => {
+    const qsa = <T extends HTMLElement>(selectors: string) =>
```

Comment:
better var name?

Reply:
it should be okay to use simple local name in short function for reduce byte used...

Reply:
🤔reduce bytes used???

Reply:
reduce bytes used????????????????

Reply:
please don't ever use such confusing variables with confusing reasons.